package.json

```diff
@@ -157,6 +157,7 @@
     "just-typeof": "^2.0.0",
     "mqtt": "^4.2.6",
     "qs": "^6.6.0",
+    "query-string": "^7.0.1",
```

Comment:
please remove this dependency, as the `qs` is doing exactly the same. Was there any reason for it? In any case we prefer `qs`, as it support everything the API suipports